steps/synopsys-pt-eco/scripts/designer-interface.tcl

```diff
@@ -0,0 +1,51 @@
+#=========================================================================
+# designer_interface.tcl
+#=========================================================================
+# The designer_interface.tcl file is the first script run by PTPX (see the
+# top of ptpx.tcl). It is the interface that connects the scripts with
+# the following:
+#
+# - ASIC design kit
+# - Build system variables
+#
+# Author : Kartik Prabhu
+# Date   : September 21, 2020
+
+
+#-------------------------------------------------------------------------
+# Parameters
+#-------------------------------------------------------------------------
+
+set pt_design_name              $::env(design_name)
+set pt_setup_margin             $::env(setup_margin)
+set pt_hold_margin              $::env(hold_margin)
+
+#-------------------------------------------------------------------------
+# Libraries
+#-------------------------------------------------------------------------
+
+set adk_dir                       inputs/adk
+
+set pt_additional_search_path   $adk_dir
+set pt_target_libraries         stdcells.db
+
+set pt_extra_link_libraries     [join "
+                                      [glob -nocomplain inputs/*.db]
+                                      [glob -nocomplain inputs/adk/*.db]
+                                  "]
```

Comment:
I hate to butt in...and feel free to ignore my comment if it's out of place...but I *strongly* recommend that whenever you do a `glob` you also add an `lsort` to order the results, otherwise the list has unpredictable / non-deterministic order...this has caused us no end of trouble in the past...it's a simple fix...the new code would look something like
```
set pt_extra_link_libraries     [join "
                                      [lsort [glob -nocomplain inputs/*.db]]
                                      [lsort [glob -nocomplain inputs/adk/*.db]]
                                  "]

set pt_adk_tcl                  $adk_dir/adk.tcl
```
...and then similarly for the code further below...
```
set pt_gl_netlist               [lsort [glob -nocomplain inputs/*.vcs.v]]
set pt_sdc                      [lsort [glob -nocomplain inputs/*.pt.sdc]]
set pt_spef                     [lsort [glob -nocomplain inputs/*.spef.gz]]
```
...also see
`cadence-genus-synthesis/scripts/designer-interface.tcl`
`cadence-genus-genlib/scripts/set_libs.tcl`

Reply:
Thanks for the feedback, I've updated it.

I'm actually not a fan of using glob (we've gotten messed up because of it too), and always end up overriding the designer-interface.tcl scripts to be explicit about the libraries/files that are to be used.